content/en/blog/_posts/grokkin-the-docs.md

```diff
@@ -0,0 +1,449 @@
+---
+layout: blog
+title: "Grokkin' the Docs"
+date: 2019-09-04
+slug: Grokkin-the-Docs
+---
+
+**Author:** [Aimee Ukasick](https://www.linkedin.com/in/aimee-ukasick/), Independent Contributor
+
+{{< figure
+    src="/images/blog/grokkin-the-docs/grok-definition.png"
+    alt="Definition of grok from Mirriam Webster online dictionary"
+    title="Definition courtesy of Mirrian Webster online dictionary"
+>}}
+
+## Intro - Experiences of a new SIG Docs contributor
+
+Brimming with excitement, I began working with the SIG Docs community in
+the beginning of August. One of the things I've been asked to focus on and
```

Comment:
You might want to add:
```suggestion
the beginning of August 2019. One of the things I've been asked to focus on and
```
(your choice of course)

Reply:
I went back and forth several times debating on putting the year in, and I eventually left it off since the blog post will be in the 2019. Now it's back in.